app/stac_api/models.py

```diff
@@ -41,6 +41,35 @@
 # 'SRID=4326;POLYGON ((5.96 45.82, 5.96 47.81, 10.49 47.81, 10.49 45.82, 5.96 45.82))'
 BBOX_CH = str(_BBOX_CH)
 
+SEARCH_TEXT_HELP_ITEM = '<div class=search-usage> Search Usage: <ul> ' \
```

Comment:
This could be wrapped by _ 
from django.utils.translation import gettext_lazy as _ for the future, if multilangual

Reply:
This is a good question for @boecklic : do we want multilingual support for the admin pages in the future or not ?

Reply:
If it doesn't cost anything to be prepared for multilingual, I'd go for it, even if I think that this kind of admin interfaces don't need to be translated

Reply:
I put it on the help_text argument in the models, since all other models with help texts have them there. (line 332 and 509)